### DISCUSSION TITLE
Commits not showing in activity

### DISCUSSION THREAD

**Discussion by JoaoGH** [community/community#102640] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have a repository where I put my old codes from the time I learned to code.
But my commits are not showing up, my credentials are correct.

Here is the [repository ](https://github.com/JoaoGH/retro-code) and here is a screenshot of a PR and my activity
![image](https://github.com/community/community/assets/41637320/bd067deb-781f-4e58-bc5a-0fc6270210de)
![image](https://github.com/community/community/assets/41637320/a0b2bdce-ea28-4d0b-8887-d0eb2c34478e)

Does anyone have any idea why this happens??

**Comment by ravindudil5han** [comment]
If your commits are not showing up in your GitHub repository, consider the following troubleshooting steps:

1. **Check Repository and Branch:**
   Ensure you are looking at the correct repository and branch where you made the commits. Sometimes, changes may be on a different branch.

2. **Committer Email Address:**
   Verify that the email address associated with your commits matches the email address linked to your GitHub account. GitHub associates commits with the account based on the commit email.

3. **Commit Timestamps:**
   Check the timestamps of your commits. If they are too old or in the future, they might not appear in the default view. GitHub orders commits by timestamp.

4. **Contributions Not Counted:**
   GitHub's contribution graph and statistics may take some time to update. If the commits are recent, wait a while and check again.

5. **Visibility Settings:**
   Ensure your commits are not part of a private branch or repository. Check the repository and branch settings to confirm visibility.

6. **Check Commit Status:**
   Review the status of your commits by navigating to the "Commits" tab in your repository. GitHub might provide additional information about any issues.

If the issue persists, it might be helpful to provide more specific details about the problem or reach out to GitHub support for assistance.

**Comment by er1927** [comment]
Hey, JoaoGH.
I'm having the same problem-already checked my credentials, waited 24 hours, and the rest of the usual recommendations. 
In my case, I only see a contribution if I make a new repository, but the 20+ commits I've added this last week don't.
Commenting with the hopes we get some traction and eventually find a way to solve this.

**Comment by JoaoGH** [comment, marked as answer]
Answering myself, commits only count when you do them on protected branches. And in my example I did squash commit on the merge, this is the reason why it only counts 1 commit
